app/models/synchronization/member.rb

```
@@ -385,6 +389,8 @@ def log
         log_attributes.merge(user_id: user.id) if user
 
         @log = CartoDB::Log.where(log_attributes).first
+      rescue e
```

I'd let that bubble up unless you're 100% this is not being logged for some obscure reason